src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -78,77 +55,16 @@ const InnerForm: React.FC<FormikProps<FormValues>> = props => {
         </Box>
       </Box>
       <Box mt={4}>
-        <Serif weight="semibold" size="4t">
-          Billing Address
-        </Serif>
         <Box mt={2}>
-          <Input
-            onBlur={handleBlur}
-            onChange={handleChange}
-            error={touched.street && errors.street}
-            required
-            title="Address"
-            name="street"
-          />
-        </Box>
-        <Box mt={2}>
-          <Serif size="3t" mb={0.5}>
-            Country*
-          </Serif>
-          <CountrySelect
-            selected={values.country}
-            onSelect={value => {
-              setFieldValue("country", value)
-              setFieldTouched("country")
+          <AddressForm
+            value={values.address}
+            onChange={(address, _key) => {
+              setFieldValue("address", address)
```

Comment:
We're dropping down to Formik's `setFieldValue` function to set the entire address as provided by the `AddressForm` component. We don't really care what changed specifically. We just want the current state.